src/appengine/private/components/upload-testcase/upload-testcase.html

```diff
@@ -261,7 +264,10 @@
     <div id="mainContent">
       <div class="toolbar">
         <div class="permission">[[getPermissionText(params)]]</div>
-        <div class="buttons"><paper-button raised toggles active="{{uploadFormOpened}}">Upload</paper-button></div>
+        <div class="buttons">
+          <paper-button raised toggles active="{{uploadFormOpened}}">Upload</paper-button>
+          <paper-button raised toggles active="{{uploadFormSimplifiedOpened}}">Quick Upload</paper-button>
```

Comment:
We need to make this conditionally show up only being a chromium deployment. You can use this: https://github.com/google/clusterfuzz/blob/910f08b9316a729c4c6b05ed260f97d1d03c3d88/src/appengine/handlers/upload_testcase.py#L241

Reply:
oh yesss will do!